__tests__/endpoints.test.js

```diff
@@ -215,9 +215,33 @@ describe('For adding a comment to a specified article', () => {
             expect(body).toMatchObject({ status: 401, msg: 'Unauthorized' })
         });
     })
+
+
+    
 })
 
+describe('Patches the article by updating the number of votes', () => {
+    test('Status 201: succesfully patches the article when a valid article is chosen and updates the number of votes when given amount to update them',()=>{
+        return request(app)
```

Comment:
This should be a 200, we are only using 201 when specifically creating a new resource